swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
 class Item:
-    pass
+    def __init__(self, category=None, condition=None):
+        self.category = category if category is not None else ""
+        self.condition = condition if condition is not None else 0
+
+
+    def __str__(self):
+        return "Hello World!"
+
+
+    def condition_description(self):
```

Comment:
What would happen if we tried to get the description for an Item with a condition of 3.5 (there are tests that set the condition to 3.5, but don't try to read the description). The description test checks for a condition of 1 and a description of 5, but we should still make sure our code covers any other reasonable conditions, including floating point values (since we've seen them used in other tests).

How could we modify this (slightly) to cover the values between the whole numbers?